package n1exercici3;

public class ExceptionCustom {
    public void getException(){
        throw new ArrayIndexOutOfBoundsException("Exepci√≥: ArrayIndexOutOfBoundsException");
    }
}
